css/style.css

```diff
@@ -0,0 +1,235 @@
+html {
+  background: #0c0c0c;
+  font-family: sans-serif;
+  font-size: 10px;
+}
+
+*{
+  box-sizing: border-box;
+}
+
+a{
```

Comment:
After the colon in the rules put one space.
See more in this guide: http://codeguide.academy/html-css.html#html-language